### ISSUE TITLE
License issue in the case of jobs parallelisation 

### ISSUE THREAD

**Issue by vd-dragan21** [milaboratory/mixcr#720]
## Checklist before submitting the issue:

 - [x] The issue is strongly related to the MiXCR software
 - [x] The issue can be reproduced with [the most recent version](https://github.com/milaboratory/mixcr/releases) of MiXCR
 - [x] There is no answer to the question in the [official documentation](http://mixcr.readthedocs.io/) and there is no duplicate issue in the [bug tracker](https://github.com/milaboratory/mixcr/issues?utf8=✓&q=)  
 - [x] Inspection of raw alignments with [exportAlignmentsPretty](http://mixcr.readthedocs.io/en/master/export.html#exporting-well-formatted-alignments-for-manual-inspection) shows that data has the expected architecture, and sample preparation artefacts are not the reason of the problem (if this is the matter of the issue)

## Expected Result

Hi! I'm using mixcr 4.0 and the offline license. 
I have a shell wrapper script that iterates through all my samples and submits sh script with mixcr command for each of them to the cluster. 
I expected several parallel jobs to be submitted at the same time, and so, several samples to be analyzed at the same time. 

## Actual Result

If submitting several jobs right after each other, like 5 samples at the same time, then only 1 is being actually analyzed and for 4 others, the license error (see below) occurred and I had to restart them. 
After playing around and testing some theories we found out that the license error occurs only if several samples are submitted with a time interval of less than 1 minute. So, after adding to the submission script extra "sleep 1m" command after each bsub command, the error didn't occur anymore. 
So, we could fix the problem for us but I wanted to let you know that this is happening!  

## Exact MiXCR commands 

Any commands, for instance, "mixcr analyze shotgun" with arguments.

## MiXCR report files 

Output (license error) when the jobs are submitted at the same time for all samples except for 1 sample:

```
java.lang.RuntimeException: Failed to acquire a lock.
	at com.milaboratory.milm.KVFilesInFolderStore.acquireLockOrError(KVFilesInFolderStore.java:120)
	at com.milaboratory.milm.KVFilesInFolderStore.lock(KVFilesInFolderStore.java:88)
	at com.milaboratory.milm.LM$ManagerThread.run(LM.java:416)
java.lang.RuntimeException: UnknownError
	at com.milaboratory.milm.LM.reportError(LM.java:307)
	at com.milaboratory.milm.LM.access$300(LM.java:25)
	at com.milaboratory.milm.LM$ManagerThread.run(LM.java:543)
License manager thread died.
License error: UnknownError
```

Please, let me know if the problem is not clear and I will try to explain it better :)

**Comment by dbolotin** [comment]
Thanks for reporting! Sorry for inconvenience! I will soon send you the fixed version.

**Comment by dbolotin** [comment]
Hi, please try [this build](https://cdn.milaboratory.com/software/mixcr/mixcr-4.0.0-27-develop-03bac154c210ea66e5afb4d1b66d94d2c0730e8d.zip). There now must be no contention between mixcr instances running in parallel.

Please, beware, this build is from the development branch, and contains many other fixes and enhancements that will be officially release with 4.1 release. Important change compared to 4.0 is a significant rework of consensus assembly algorithm, which may produce significantly different results for noisy single-cell or UMI data. Detailed description of changes will be available with mixcr 4.1.
